config/config.go

```diff
@@ -76,13 +76,22 @@ type (
 		JwksUri                       string `json:"jwks_uri"`
 	}
 
+	ServerType string
+
 	TokenOperation int
 
 	TokenGenerationOpts struct {
 		Operation TokenOperation
 	}
 )
 
+const (
+	CacheType    ServerType = "Cache"
```

Comment:
Switch these to a bitmask.  We'll eventually want these to be a multi-selection (e.g., call `InitServer` for both origin and director).